examples/examples_tags.json

```diff
@@ -0,0 +1,5 @@
+{
+	"core_basic_window": ["window"],
+	"core_basic_screen_manager": ["screen", "scene"],
+	"core_2d_camera": ["2d", "camera", "zoom"]
+}
```

Comment:
i did make a few examples so you can see the concept and structure. everything is required to be lowercase but otherwise anything can be put in here and the search will pick it up

the error handler will prevent invalid syntax from breaking the functionality

Reply:
If you download this branch to try it out, you can test/add/change the tags to verify it works and get a better idea of the concept